packages/amplify-category-auth/provider-utils/awscloudformation/triggers/UserMigration/function-template-dir/trigger-index.js

```diff
@@ -0,0 +1,13 @@
+/*
+  this file will loop through all js modules which are uploaded to the lambda resource,
+  provided that the file names (without extension) are included in the "MODULES" env variable.
+  "MODULES" is a comma-delimmited string.
+*/
+
+exports.handler = (event, context, callback) => {
+  const modules = process.env.MODULES.split(',');
+  for (let i = 0; i < modules.length; i += 1) {
+    const { handler } = require(modules[i]);
```

Comment:
To support node v10.x.x, I think we'll need to change this to:
`    const { handler } = require(`./${modules[i]}`);`